dashboards/vmagent.json

```diff
@@ -3691,6 +3691,108 @@
           ],
           "title": "Restarts ($job)",
           "type": "timeseries"
+        },
```

Comment:
this dashboard isn't synced to `vm` type

Reply:
what do you mean?

Reply:
I think `make dashboards-sync` wasn't called.